opentelemetry-sdk/src/opentelemetry/sdk/trace/sampling.py

```diff
@@ -67,28 +69,47 @@
 
 
 class Decision:
```

Comment:
> Decision is an enum.

Any reason why we wouldn't be using an `Enum` in that case?

```suggestion
class Decision(Enum):
```

And if if was an `Enum`, I think it would also be nicer to compare by identity instead of equality.